src/graphql/query/myKiva.graphql

```diff
@@ -79,5 +79,38 @@ query myKivaQuery {
       id
       preferences
     }
+    transactions(
```

Comment:
Is this different data from the transaction updates?

Reply:
Probably related, but better approach as we are already prefetching user data, so including this is simplest solution.